app/util/appUtil.js

```diff
@@ -0,0 +1,50 @@
+const { ApiPromise, WsProvider } = require('@polkadot/api')
```

Comment:
if we're not including the node in the template, this whole file can go + envs for node config + polkadot/api package